torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -55,9 +50,7 @@ def __init__(self, mode, input_size, hidden_size,
             raise ValueError("Unrecognized RNN mode: " + mode)
 
         self._all_weights = []
```

Comment:
Maybe we can change `_all_weights` to `_all_weights_names`? This is more consistent with `_all_weight_values`.